interface-spec.md

```diff
@@ -99,6 +109,7 @@ TODO: Do we need to define a different interface, or is a quad simply a triple w
 - `.blankNode()` returns a new instance of BlankNode.
 - `.literal(String value, String language, String datatype)` returns a new instance of Literal.
 - `.variable(String name)` returns a new instance of Variable. This method is optional.
+- `.defaultGraph()` returns an instance of DefaultGraph. This method should return a singleton.
```

Comment:
Yes, but with constant someone could expect === can be used to compare it. Maybe add "an implementation must use the same constant".
Am 22.01.2016 10:28 vorm. schrieb Ruben Verborgh <notifications@github.com>:In interface-spec.md:
> @@ -99,6 +109,7 @@ TODO: Do we need to define a different interface, or is a quad simply a triple w
>  - `.blankNode()` returns a new instance of BlankNode.
>  - `.literal(String value, String language, String datatype)` returns a new instance of Literal.
>  - `.variable(String name)` returns a new instance of Variable. This method is optional.
> +- `.defaultGraph()` returns an instance of DefaultGraph. This method should return a singleton.

Right, but within a library, it must be a constant?

—Reply to this email directly or view it on GitHub.